features/step_definitions/basic_steps.rb

```diff
@@ -18,6 +12,14 @@
     click_on link
 end
 
+Given("the following dishes exist") do |table|
+    table.hashes.each do |dish|
+        dish_category = Category.create(name: dish[:category])
```

Comment:
can you guys try using `dish_category = Category.find_or_create_by(name: dish[:category])` instead on line 17

Reply:
Done